core/controllers/editor.py

```diff
@@ -117,7 +117,14 @@ def put(self, exploration_id):
         """Updates properties of the given exploration."""
         exploration = exp_fetchers.get_exploration_by_id(exploration_id)
         version = self.payload.get('version')
-        _require_valid_version(version, exploration.version)
```

Comment:
Why remove this function?

Reply:
This function use to return an error whenever there was a version mismatch, but now as we are handling the cases when the changes are done from the old version to the new version, I removed this function.